### ISSUE TITLE
Release 4.9.0 - Alpha 3 - Vulnerability Detection E2E tests

### ISSUE THREAD

**Issue by rauldpm** [wazuh/wazuh#24809]
## Tests information

|                                      |                                            |
|--------------------------------------|--------------------------------------------|
| **Main release stage issue**     |            https://github.com/wazuh/wazuh/issues/24714                   |
| **Main Vulnerability Detection tests issue**     |           https://github.com/wazuh/wazuh/issues/24808                       |
| **Version**                          |                   4.9.0                         |
| **Release stage**              |                   Alpha 3                      |
| **Tag**                              |          https://github.com/wazuh/wazuh-qa/tree/v4.9.0-alpha3                               |
| **Previous E2E Vulnerability tests issue**      |       https://github.com/wazuh/wazuh/issues/24594                        |

### Description

The following issue aims to run all [Vulnerability Detector E2E](https://github.com/wazuh/wazuh-qa/issues/4369) tests for the current release candidate, report the results, and open new issues for any encountered errors.

### Instruction

Run the vulnerability detector tests using the [Test_e2e_system](https://ci.wazuh.info/job/Test_e2e_system/) pipeline

### Conclusion :red_circle:
**New issues**

- https://github.com/wazuh/wazuh-qa/issues/5609
- https://github.com/wazuh/wazuh-qa/issues/5608

**Known issue**
- https://github.com/wazuh/intelligence-platform/issues/1467

**Comment by Rebits** [comment]
Build: https://ci.wazuh.info/job/Test_e2e_system/327/

**Comment by Rebits** [comment]
Unexpected failure provisioning E2E environment. Reported in https://github.com/wazuh/wazuh-jenkins/issues/6795

**Comment by Rebits** [comment]
Multiple failures in checks related to mitigated alerts.
Reported in https://github.com/wazuh/wazuh-qa/issues/5608

**Comment by Rebits** [comment]
# Evidences 

Build: https://ci.wazuh.info/job/Test_e2e_system/331/
Report:  [Test_e2e_system_331_test_vulnerability_detector.zip](https://github.com/user-attachments/files/16362677/Test_e2e_system_331_test_vulnerability_detector.zip)

# Analysis

## Unexpected error

- Solved in 4.10.0 but not in 4.9.0. Issue to be opened. 
```
            "2024/07/23 12:29:38 wazuh-agentd: WARNING: (4101): Waiting for server reply (not started). Tried: '172.31.3.87'. Ensure that the manager version is 'v4.9.0' or higher."
```


- Missing vulnerabilities
```
CVE-2022-0778
```

**Comment by pro-akim** [comment]
## Review Notes
LGTM
